examples/qt_types_standalone/Cargo.toml

```diff
@@ -18,7 +18,7 @@ crate-type = ["staticlib"]
 [dependencies]
 cxx = "1.0"
 cxx-qt = { path = "../../cxx-qt" }
-cxx-qt-lib = { path = "../../cxx-qt-lib" }
+cxx-qt-lib = { path = "../../cxx-qt-lib", features=["Qt6"] }
```

Comment:
check if feature is the best way or if we can have something similar to defines work in C++.

eg can we replicate #[cfg(target_os = "macos")] with --cfg 'qt_major_version="5"' at the rustc level

https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg-attribute
https://doc.rust-lang.org/rustc/command-line-arguments.html#--cfg-configure-the-compilation-environment